camera_control/camera_control/camera_node_client.py

```diff
@@ -0,0 +1,65 @@
+import rclpy
+
+from std_srvs.srv import Empty
+from camera_control_interfaces.srv import GetBool, GetPosData
+from laser_control_interfaces.msg import Point
+
+
+# Could make a mixin if desired
+class CameraNodeClient:
+    def __init__(self, node):
+        node.camera_get_lasers = node.create_client(
```

Comment:
do these need to be on the node, or can it be on self?

Reply:
It need to be on the node itself in this implementation. We could make a class that a rod node inheritances that could let us use self. That node would then have access to all the client functions without needing a client object

Reply:
I found this method easier in understanding what’s going on without having to look under the hood so to speak